CrowsZA/functions/fn_togglePathing.sqf

```diff
@@ -20,18 +20,7 @@ if(count _units == 0 || _action < 0 || _action > 2) exitWith { false };
 
 {
 	if(isPlayer _x) then { continue };
-
-	switch(_action) do {
-		case 0: { _x disableAI "PATH"; };
-		case 1: { _x enableAI "PATH"; };
-		default {
-			if(_x checkAIFeature "PATH") then {
-				_x disableAI "PATH";
-			} else {
-				_x enableAI "PATH";
-			}
-		};
-	};	
+	[_x, _action] remoteExec ["crowsZA_fnc_toggleUnitPathing", _x];
```

Comment:
The problem I see here is following. 

If you have a client that does not have this mod loaded, (clientside, so maybe only one zeus have it loaded), you run into the problem of trying to call a function that does not exist on that zeus, if that zeus own the unit. Same if the "headless" is not running the mod. As it will try to execute the function ``crowsZA_fnc_toggleUnitPathing`` that doesn't exist on that client. 

You should remote-exec the built-in function instead. So ``disableAI`` and ``enableAI``. So do the logic here, and remote exec that action you want. Does require to have two remoteExecs based on if you need disable or enable. E.G.

```
[_x, "PATH"] remoteExec ["disableAI", _x];
```

Reply:
D'oh - should be resolved now